lib/galaxy/datatypes/binary.py

```diff
@@ -1920,7 +1920,6 @@ class Dcd(Binary):
     False
     """
     file_ext = "dcd"
-    edam_format = "format_3842"  # molecular simulation data
```

Comment:
You can really use `edam_data` it is supported by galaxy. See here: https://github.com/galaxyproject/galaxy/blob/30e3658b8b0e2f6b975dc6ccccb0cc8cc040247c/lib/galaxy/datatypes/annotation.py#L15